charts/DataTab.tsx

```diff
@@ -45,7 +45,7 @@ export class DataTab extends React.Component<{
                 const row = [
```

Comment:
TypeScript incorrectly infers the type of `row` now that the `formatYearFunction` is used.

The previous inference for `row` was `(string | number)[]`, because `year` was a `number`. The new inference is `string[]`. In the code lower below, when the rest of the columns (the variable values) are added, the `string[]` inference doesn't work.

One way to fix this is to set the type explicitly:

```suggestion
                const row: (string | number)[] = [
```

Reply:
Ah ok, I thought we had agreed not to modify references to `year`, but now I see you may have been referring just to actual code references, not the string used for the row title. I will address this.

Wrt to the TypeScript reference, good to know! I haven't encountered this type of TypeScript error before. I'll look into it to find best practices.